platform/mesos/executor.go

```diff
@@ -238,7 +238,7 @@ func (ec *Executor) Run(c context.Context) {
 			log.Infof("gracefully exiting because framework checkpointing is NOT enabled")
 			return
 		}
-		if time.Now().Sub(disconnected) > ec.cfg.RecoveryTimeout {
+		if time.Since(disconnected) > ec.cfg.RecoveryTimeout {
```

Comment:
Is there any effects of walltime or monotonic time ?

Reply:
`time.Since` is just a shorthand for `time.Now().Sub` and suggested by golint.